src/view/com/pager/TabBar.tsx

```diff
@@ -139,7 +140,10 @@ export function TabBar({
                 <Text
                   type={isDesktop || isTablet ? 'xl-bold' : 'lg-bold'}
                   testID={testID ? `${testID}-${item}` : undefined}
-                  style={selected ? pal.text : pal.textLight}>
+                  style={[
+                    selected ? pal.text : pal.textLight,
+                    {lineHeight: 20},
```

Comment:
`Feeds ✨ ` was increasing the line height slightly and resulting in a gap between the selected state and the top post, which looked bad. `20` appears to match well on desktop/mobile and fix the problem.